frontend/schema/class-schema-howto.php

```diff
@@ -146,7 +148,10 @@ private function add_steps( &$data, $steps ) {
 				'url'   => $schema_id,
 			);
 
-			if ( empty( $step['jsonName'] ) ) {
+			$json_text = strip_tags( $step['jsonText'], '<h1><h2><h3><h4><h5><h6><br><ol><ul><li><a><p><b><strong><i><em>' );
```

Comment:
This whitelist for `jsonText` is also used on line 193. It would be nicer to have this in a variable as per the DRY principle.